### PR TITLE
chore: add git file change tests

### PR DESCRIPTION
**Pull request by macox**
#### Description

Some tests for git HasChanges and HasFileChanged functions

### CODE REVIEW

File: pkg/gits/git_cli_test.go [hunk 1]
```diff
@@ -583,6 +583,55 @@ var _ = Describe("Git CLI", func() {
 			})
 		})
 	})
+
+	Describe("#HasChanges", func() {
```
Comment (hferentschik) [inline pkg/gits/git_cli_test.go]: :-) Yeah

**Comment by abayer** [comment]
/hold
holding briefly to avoid batch hell

**Pull request merged**